statsmodels/distributions/discrete.py

```diff
@@ -22,3 +22,28 @@ def _pmf(self, x, mu, alpha, p):
 
 genpoisson_p = genpoisson_p_gen(name='genpoisson_p',
                                 longname='Generalized Poisson')
+
+class zipoisson_gen(rv_discrete):
+    '''A Zero Inflated Poisson distribution
+    '''
+    def _argcheck(self, mu, w):
+        return True
```

Comment:
either this is redundant or should be something like
`return (mu > 0) & (w >= 0) & (w<=1)`